File: Aula07/testeFE.py
# print ("Digite um número: ")
# celsius = float(input())

# if ((celsius >= 0) and (celsius <= 1000)):
#     fahrenheit = (celsius * 1.80) + 32.00
#     kelvin = celsius + 273.15

#     ans = [float(format(kelvin,".5f")), float(format(fahrenheit, ".5f"))]

# print (ans)


celsius = float(input())

if ((celsius >= 0) and (celsius <= 1000)):
    fahrenheit = (celsius * 1.80) + 32.00
    kelvin = celsius + 273.15

    ans = [float(kelvin), float(fahrenheit)]

    print(f"{ans[0]:.5f}, {ans[1]:.5f}")

File: Aula07/testeFE.js
var input = readline();
var valor = Number(input);

if (valor >= 0 && valor <= 1000) {
    var fahrenheit = (celsius * 1.80) + 32.00;
    var kelvin = celsius + 273.15;

    ans = [kelvin.toFixed(5), fahrenheit.toFixed(5)]
}


